Remove config profile flags from pcr and update responses from show virtual cluster (#18488)
cockroachdb/docs: src/current/v24.1/set-up-physical-cluster-replication.md

- All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Copy certificates](#step-3-copy-certificates).
- An [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) on the primary **and** standby clusters. You must use the system virtual cluster on the primary and standby clusters to enable your {{ site.data.products.enterprise }} license.
- The primary and standby clusters **must have the same [region topology]({% link {{ page.version.version }}/topology-patterns.md %})**. For example, replicating a multi-region primary cluster to a single-region standby cluster is not supported. Mismatching regions between a multi-region primary and standby cluster is also not supported.

## Step 1. Create the primary cluster

### Initialize the primary cluster

> To enable physical cluster replication, it is necessary to initialize the CockroachDB cluster with the appropriate flag to determine the primary and standby clusters.

Reviewer: Maybe: "to determine the primary and standby cluster" to "create the appropriate virtual clusters on the primary and standby cluster"

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    cockroach sql --url \
    "postgresql://root@{node IP or hostname}:26257?options=-ccluster=system&sslmode=verify-full" \
    --certs-dir "certs"
    ~~~

    The prompt will include `system` when you are connected to the system virtual cluster.

    {{site.data.alerts.callout_info}}

>     You should only connect to the system virtual cluster for cluster administration. To work with databases, tables, or workloads, connect to the virtual cluster.

Author: No, it doesn't need a qualifier necessarily. Users can actually create virtual clusters if they want and change the names of them, I will change this to "a" so it is indefinite.

    SET CLUSTER SETTING enterprise.license = 'your enterprise license';
    ~~~

> 1. Set the `kv.rangefeed.enabled` cluster setting to `true`. The replication job connects to a long-lived request, a _rangefeed_, which pushes changes as they happen:

Reviewer: This is true, but we probably want to enable it on both. 

Perhaps we should add this to the work that our init flags do this. I've opened a PR that adds this to the work the init flags do. We'll see how that discussion goes.

> 1. Set the `kv.rangefeed.enabled` cluster setting to `true`. The replication job connects to a long-lived request, a _rangefeed_, which pushes changes as they happen:

Author: OK, for now, I will add this same step into the standby section too. I also ran into needing to enable for the standby while I have been testing fast failback.

    {% include_cached copy-clipboard.html %}
    ~~~ sql
    SET CLUSTER SETTING kv.rangefeed.enabled = true;
    ~~~

1. Confirm the status of your virtual cluster: {% comment %}Link to SQL ref page here{% endcomment %}

    {% include_cached copy-clipboard.html %}
    ~~~ sql
    SHOW VIRTUAL CLUSTERS;
    ~~~

    The output will include the `system` virtual cluster and the `main` virtual cluster:

    ~~~
      id |  name  | data_state | service_mode
    -----+--------+------------+---------------
       1 | system | ready      | shared

>        3 | main   | ready      | shared

Author: I don't know if this really matters, but the newly created `main` default VC always creates at `id 3`... which seemed like an artifact of the `template` VC being there.

Reviewer: We do this on purpose because we _may_ want to use `id 2` for something special in the future.

Author: Ooooo ok! That sounds interesting, can't wait to find out what it is!! 😄 🍿

## Step 2. Create the standby cluster

### Initialize the standby cluster

> Similarly to the primary cluster, you must initialize the standby cluster with the `--virtualized-empty` flag. This creates a _virtualized cluster_ with a system virtual cluster.

Reviewer 2: ```suggestion
While you initialized the primary cluster with the `--virtualized` flag, you must initialize the standby cluster with the `--virtualized-empty` flag. This creates a _virtualized cluster_ with only a system virtual cluster.
```

For additional detail on the standard CockroachDB connection parameters, refer to [Client Connection Parameters]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url).

Cluster | Virtual Cluster | Usage | URL and Parameters
--------+-----------+-------+------------+----
Primary | System | Set up a replication user and view running virtual clusters. Connect with [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}). | `"postgresql://root@{node IP or hostname}:26257?options=-ccluster=system&sslmode=verify-full"`<br><br><ul><li>`options=-ccluster=system`</li><li>`sslmode=verify-full`</li></ul>Use the `--certs-dir` flag to specify the path to your certificate.

> Primary | Main | Add and run a workload with [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}). | `"postgresql://root@{node IP or hostname}:{26257}?options=-ccluster=main&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"`<br><br>{% include {{ page.version.version }}/connect/cockroach-workload-parameters.md %} As a result, for the example in this tutorial, you will need:<br><br><ul><li>`options=-ccluster={virtual_cluster_name}`</li><li>`sslmode=verify-full`</li><li>`sslrootcert={path}/certs/ca.crt`</li><li>`sslcert={path}/certs/client.root.crt`</li><li>`sslkey={path}/certs/client.root.key`</li></ul>

Reviewer 2: ```suggestion
Primary | Main | Add and run a workload with [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}). | `"postgresql://root@{node IP or hostname}:{26257}?options=-ccluster=main&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"`<br><br>{% include {{ page.version.version }}/connect/cockroach-workload-parameters.md %} As a result, for the example in this tutorial, you will need:<br><br><ul><li>`options=-ccluster=main`</li><li>`sslmode=verify-full`</li><li>`sslrootcert={path}/certs/ca.crt`</li><li>`sslcert={path}/certs/client.root.crt`</li><li>`sslkey={path}/certs/client.root.key`</li></ul>
```

In my understanding, the non-system virtual cluster is called `main` and the user cannot control the name of it, so a variable `{virtual_cluster_name}` is not necessary.  Or is this placeholder meant to possible hold either `system` or `main`?

Author: The user can control the name of virtual clusters.
